iceoryx2-cli/lib/src/cli.rs

```diff
@@ -21,16 +27,22 @@ pub fn help_template(cli_name: &str, show_external_commands: bool) -> String {
         "Options:".bright_green().bold(),
     );
 
-    if show_external_commands {
-        template.push_str(&format!(
-            "\n\n{}\n{{subcommands}}\n{}{}",
-            "Commands:".bright_green().bold(),
-            "  ...            ".bold(),
-            "See external installed commands with --list"
-        ));
-    } else {
-        // Only add subcommands section without the "Commands:" header
-        template.push_str("\n\n{{subcommands}}");
+    match command_help {
+        HelpOptions::PrintCommandSection | HelpOptions::PrintHintForInstalledCommands => {
+            template.push_str(&format!(
+                "\n\n{}\n{{subcommands}}",
+                "Commands:".bright_green().bold(),
+            ));
```

Comment:
It seems `PrintHintForInstalledCommands` is only used with `iox2`. Will there ever be commands available for `iox2 --help`?

If not, what do you think about splitting these options like it was before with:
```rs
HelpOptions::PrintHintForInstalledCommands => {
    template.push_str(&format!(
        "\n\n{}\n{}",
        "Commands:".bright_green().bold(),
        "[See external installed commands with --list]".bright_yellow(),
    ));
}
```

Reply:
> Will there ever be commands available for iox2 --help?

It's anyones guess ! 😆 The CLI is mimics `cargo` though, which does have in-built commands.

> what do you think about splitting these options like it was before with ...

Why ?

Reply:
The difference to cargo is that it shows a limited amount of commands and the ellipsis with the hint is the last entry.
Here, there are no commands shown but an empty line between `Commands` and the line with the ellipsis. Therefore, I thought it might make sense to put more emphasis on the hint.

Reply:
@elBoberido If you would really like to have this, could you add this to (and reopen the issue if its closed): https://github.com/eclipse-iceoryx/iceoryx2/issues/646

I will get it in the next (inevitable) clean-up round.

Reply:
@orecham it was just a suggestion and the reasoning behind it. No need to sped more time on it